ooiservices/app/redmine/routes.py

```diff
@@ -187,14 +184,15 @@ def get_redmine_users():
                         status=400,
                         mimetype="application/json")
 
-    proj = request.args['project']
-    project = redmine.project.get(proj).refresh()
-
+    all_users = redmine.user.all(offset=1, limit=100)
```

Comment:
Yeah, that should probably get replaced with a pagination.  But yea, we should return all.